index.go

```diff
@@ -226,9 +231,9 @@ func main() {
 		r.Get("/user/:id", customer_ctlr.GetUserById)
 		r.Post("/user/:id", customer_ctlr.UpdateCustomerUser)
```

Comment:
For this, I can't find any instance of it in the logs, but looking at the code it seems like they might be intended to use their own key.